claudiu-website-main-page.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Claudiu Mantu</title>
+    <link rel="stylesheet" href="claudiu-website-main-page.css">
+
+</head>
+
+
+
+<body>
+
+    <header>
+        <nav class="container">
+            <div class="main">
+                <div class="logo">
+                    <img src="manta-logo-2.png" alt="Claudiu logo">
+                </div>
+                <ul>
+
+                    <li><a href="contact-page.html">Contact</a></li>
+                    <li><a href="fun-facts.html">Fun Facts</a>
+                    </li>
+                    <li class="active"><a href="claudiu-website-main-page.html">Home</a>
+                    </li>
+                </ul>
+            </div>
+
+        </nav>
+    </header>
+
+    <section class="intro">
+        <div class="title">
+            <p><em>Hey, I'm</em></p>
+            <h1><strong>Hey, I'm Claudiu
+                    & I'm an aspiring Web Developer</strong></h1>
+        </div>
+
+        <div class="button">
+            <a href="https://drive.google.com/file/d/1bm87Y9GxolY4If4XvqmOqET1TTJx3G1W/view?usp=sharing"
+                class="btn">LEARN
+                MORE ABOUT ME</a>
+
+        </div>
+
+    </section>
+
+    <section class="interests">
+        <div class="title">
+            <p><em>Hey, I'm</em></p>
+            <h1><strong>Hey, I'm Claudiu
+                    & I'm an aspiring Web Developer</strong></h1>
+        </div>
+
+        <div class="button">
+            <a href="fun-facts.html" class="btn">LEARN
+                MORE ABOUT ME</a>
+
+        </div>
```

Comment:
Duplication of code.

```suggestion
    <section class="intro">
        <div class="title">
            <p><em>Hey, I'm</em></p>
            <h1><strong>Hey, I'm Claudiu
                    & I'm an aspiring Web Developer</strong></h1>
        </div>

        <div class="button">
            <a href="https://drive.google.com/file/d/1bm87Y9GxolY4If4XvqmOqET1TTJx3G1W/view?usp=sharing"
                class="btn">LEARN
                MORE ABOUT ME</a>

        </div>

    </section>

    <section class="interests">
        <div class="title">
            <p><em>Hey, I'm</em></p>
            <h1><strong>Hey, I'm Claudiu
                    & I'm an aspiring Web Developer</strong></h1>
        </div>

        <div class="button">
            <a href="fun-facts.html" class="btn">LEARN
                MORE ABOUT ME</a>

        </div>
```